frontend/packages/integration-tests-cypress/tests/i18n/pseudolocalization.spec.ts

```diff
@@ -14,38 +20,171 @@ describe('Localization', () => {
     cy.logout();
   });
 
-  it('pseudolocalizes masthead', () => {
-    cy.log('test masthead');
-    cy.visit('/dashboards?pseudolocalization=true&lng=en');
-    masthead.clickMastheadLink('help-dropdown-toggle');
-    cy.byTestID('application-launcher-item').isPseudoLocalized();
-  });
+  const tableColumnHeaders =
+    'th > .pf-c-table__button > .pf-c-table__button-content > .pf-c-table__text';
+  const sectionHeadings = '[data-test-section-heading]';
 
-  it('pseudolocalizes navigation', () => {
-    cy.log('test navigation');
-    cy.visit('/dashboards?pseudolocalization=true&lng=en');
-    cy.byTestID('nav').isPseudoLocalized();
-  });
+  const createExampleResource = (url: string) => {
+    listPage.clickCreateYAMLbutton();
+    cy.byTestID('resource-sidebar').should('exist');
+    yamlEditor.isLoaded();
+    yamlEditor.clickSaveCreateButton();
+    cy.get(errorMessage).should('not.exist');
+    cy.visit(url);
+  };
 
-  it('pseudolocalizes activity card', () => {
-    cy.log('test activity card components');
-    cy.visit('/dashboards?pseudolocalization=true&lng=en');
-    cy.byTestID('activity').isPseudoLocalized();
-    cy.byTestID('activity-recent-title').isPseudoLocalized();
-    cy.byTestID('ongoing-title').isPseudoLocalized();
-    cy.byTestID('events-view-all-link').isPseudoLocalized();
-    cy.byTestID('events-pause-button').isPseudoLocalized();
-  });
+  const deleteExampleResource = (kind: string) => {
+    detailsPage.clickPageActionFromDropdown(`Delete ${kind}`);
+    modal.shouldBeOpened();
+    modal.submit();
+    modal.shouldBeClosed();
+  };
 
-  it('pseudolocalizes utilization card', () => {
-    cy.log('test utilization card components');
-    cy.visit('/dashboards?pseudolocalization=true&lng=en');
-    cy.byTestID('utilization-card-item-text').isPseudoLocalized();
-  });
+  const testObjs = OrderedMap<
+    string,
+    {
+      url: string;
+      createResource?: boolean;
+      deleteResource?: string;
+      testIds?: string[];
+      selectors?: string[];
+    }
+  >()
+    .set('masthead', {
+      url: '/dashboards',
```

Comment:
I would pick a URL that isn't as heavy as `/dashboards` for these common components. Maybe the cluster settings page.

Reply:
fixed